### ISSUE TITLE
[Enhancement] Include temperature measurements in timeline

### ISSUE THREAD

**Issue by inducer** [babybuddy/babybuddy#662]
When a kid is having a fever, temperature measurements in the timeline would (IMO) aid in getting other caregivers "up to speed" on what has been happening on hand-overs.

**Comment by cdubz** [comment]
Makes sense. Of the measure _not_ included in the timeline that seems like the one that would actually be valuable.
